book/src/dev/rfcs/xxxx-block-subsidy.md

```diff
@@ -146,12 +144,25 @@ pub enum FundingStreamRange {
 /// as described in [protocol specification §7.9.1][7.9.1].
 /// 
 /// [7.9.1]: https://zips.z.cash/protocol/protocol.pdf#zip214fundingstreams
-const FUNDING_STREAM_HEIGHT_RANGES: &[(Network, Range)] = &[
+const FUNDING_STREAM_HEIGHT_RANGES: &[(Network, Range<_>)] = &[
     (Network::Mainnet, Height(1_046_400)..Height(2_726_400)),
     (Network::Testnet, Height(1_028_500)..Height(2_796_000)),
 ];
 ```
 
+Each `FundingStreamReceiver` for the mainnet must have 48 payment addresses associated with and 51 for the testnet. 
+
+https://zips.z.cash/zip-0214#mainnet-recipient-addresses
+
+The following constants are needed:
+
+- `FUNDING_STREAM_ECC_ADDRESSES_MAINNET: [&'static str; 48]`
+- `FUNDING_STREAM_ECC_ADDRESSES_MAINNET: [&'static str; 48]`
+- `FUNDING_STREAM_ECC_ADDRESSES_MAINNET: [&'static str; 48]`
+- `FUNDING_STREAM_ECC_ADDRESSES_TESTNET: [&'static str; 51]`
+- `FUNDING_STREAM_ECC_ADDRESSES_TESTNET: [&'static str; 51]`
+- `FUNDING_STREAM_ECC_ADDRESSES_TESTNET: [&'static str; 51]`
```

Comment:
Some of these constants should be named ZF and MG 👀

Reply:
oops, sorry, fixed that. thanks.